joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -912,10 +942,12 @@ controller_interface::CallbackReturn JointTrajectoryController::on_configure(
     std::bind(&JointTrajectoryController::goal_accepted_callback, this, _1));
 
   resize_joint_trajectory_point(state_current_, dof_);
-  resize_joint_trajectory_point_command(command_current_, dof_);
+  resize_joint_trajectory_point_command(
+    command_current_, dof_, std::numeric_limits<double>::quiet_NaN());
```

Comment:
Shouldn't this be `num_cmd_joints_`?

```suggestion
    command_current_, num_cmd_joints_, std::numeric_limits<double>::quiet_NaN());
```

Reply:
no. it has size of `dof_` and is later indexed by the `map_cmd_to_joints_`. The full vectory is then used for the status message publisher, where the non-commands remain NaN

Reply:
Got it.